tempesta_fw/http.c

```diff
@@ -713,6 +715,34 @@ tfw_http_req_error(TfwSrvConn *srv_conn, TfwHttpReq *req,
 	__tfw_http_req_error(req, equeue, status, reason);
 }
 
+static inline void
+tfw_http_error_resp_switch(TfwHttpReq *req, unsigned short status,
+			   const char *reason)
+{
+	switch(status) {
+	case 403:
+		tfw_http_send_403(req, reason);
+		break;
+	case 404:
+		tfw_http_send_404(req, reason);
+		break;
+	case 500:
+		tfw_http_send_500(req, reason);
+		break;
+	case 502:
+		tfw_http_send_502(req, reason);
+		break;
+	case 504:
+		tfw_http_send_504(req, reason);
+		break;
+	default:
+		TFW_WARN("Unexpected response error code: [%d]\n",
+			 status);
+		tfw_http_send_500(req, reason);
+		break;
+	}
+}
```

Comment:
The comment https://github.com/tempesta-tech/tempesta/issues/658#issuecomment-279260358 isn't addressed. Please load files named like `404.html` or `500.html` from etc directory and use the content as a response body. Otherwise a user just sees blank pages. Currently we load TLS certificates from the files, so it'd be good to generalize the routines and use the same code for TLS certificates and the error responses. Probably a new variables, `$TFW_PATH/etc` should be passed to tempesta_fw.ko - the path can be used for TLS certificates as well as for the error codes.

Reply:
Corrected: configurable custom error pages added.